frontend/integration-tests/views/catalog-page.view.ts

```diff
@@ -9,7 +9,7 @@ export const filterSectionFor = (group: string) => $(`[data-test-group-name=${gr
 export const showMoreFilters = (group: string) => $(`[data-test-group-name=${group}] .btn-link`);
 export const filterCheckboxFor = (id: string) => $(`input[data-test=${id}]`);
 export const clickFilterCheckbox = (id: string) => filterCheckboxFor(id).click();
-export const filterCheckboxCount = (id: string) => filterCheckboxFor(id).$('.item-count').getText()
+export const filterCheckboxCount = (id: string) => $(`input[data-test='${id}']+.item-count`).getText()
```

Comment:
definition changes

Reply:
```suggestion
export const filterCheckboxCount = (id: string) => $(`${filterCheckboxFor(id)}+.item-count`).getText()
```

Reply:
It reports invalid selector if use `$(`${filterCheckboxFor(id)}+.item-count`).getText()`
```
1) Developer Catalog : filters catalog tiles by 'Source-To-Image' Type correctly
   InvalidSelectorError: invalid selector: An invalid or illegal selector was specified
```